server/routes/notification.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { notifications } = require("../controllers/notification");
+
+router.route("/notification").get(protect, notifications);
```

Comment:
This is going in the right direction! We can drop the `notification` keyword for this route[^*], `route("/").get` will be enough to convey what this is doing by convention.

[^*]: Usually, express APIs will declare the routes for each resource in `app.js` (e.g. [check out these lines](https://github.com/hatchways/team-reindeer/blob/e28f6518c23640ee2b5743570b4a31ad19dbf156/server/app.js#L47-L49)). All you have to do is import this router in `app.js` and "use" it 👍🏼